README.md

```diff
@@ -51,6 +51,23 @@ To use kind, you will need to [install docker].
 Once you have docker running you can create a cluster with `kind create cluster`  
 To delete your cluster use `kind delete cluster`
 
+### Auto completions
+
+To enable Bash completion for Kind commands, you can use the bash-completions project:
+
+```console
+sudo apt | dnf | pacman install bash-completion
```

Comment:
users are going to cut and paste this and run into it not being a real command, so we should split it out and instead point to the upstream install docs.